doc/releases/migration-guide-4.2.rst

```diff
@@ -102,6 +102,17 @@ GPIO
   now left as a placeholder and mapper.
   The labels have also been changed along, so no changes are necessary for regular use.
 
+LED
+===
+
+* Renamed the devicetree property ``log_scale_en`` to ``log-scale-en``.
+
+PWM
+===
+
+* Renamed the devicetree property ``silabs,ch_prescaler`` to ``silabs,ch-prescaler``.
```

Comment:
I don't believe these two PWM properties need migration guide entries, they were added to the tree after 4.1 release.